nautobot_netbox_importer/diffsync/adapters/abstract.py

```diff
@@ -231,7 +231,7 @@ def add(self, obj: DiffSyncModel):
         # in case we have duplicate objects with the same unique_id but different PKs.
         modelname = obj.get_type()
         if obj.pk in self._data_by_pk[modelname]:
-            raise ObjectAlreadyExists(f"Object {modelname} with pk {obj.pk} already loaded")
+            raise ObjectAlreadyExists(f"Object {modelname} with pk {obj.pk} already loaded")  # pylint: disable=E1120
```

Comment:
Can you please disable pylint errors by label instead of by number?

This however looks to be an actual bug, not something we should disable - [`ObjectAlreadyExists.__init__`](https://github.com/networktocode/diffsync/blob/develop/diffsync/exceptions.py#L42) expects two arguments, not only one.

Reply:
OMG! where is <facepalm> emot? :)